.flake8

```diff
@@ -1,7 +1,6 @@
 [flake8]
 ignore =
   E501  # line too long, defer to black
-  F401  # unused import, defer to pylint
```

Comment:
is pylint not doing the work here? according to this comment that was the intent.

Reply:
It is not, we have been having lots of unused imports that pylint is not detecting. In fact, what detects the unused imports is not pylint but flake8.